routes/quiz.js

```diff
@@ -83,17 +86,39 @@ router.get('/results', function(req, res) {
           res.json({success: false});
         }
         else {
-          res.json({all_scores: req.session.scores, this_score: modified_score, success: true});
+          res.json({all_scores: req.session.scores, this_score: modified_score, success: true, percentile: req.session.percentile});
         }
       });
     });
 });
 
+/* Takes in a sorted array that includes the score and the score. Sorted position over length = percentile. 
+Am ignoring true math piece of even/odd stuff */
+var calculatePercentile = function(req, score, all_scores){
+  len = all_scores.length;
+  count = 0;
+  for (i=0;i<all_scores.length;i++){
+    if(all_scores[i]< score){
```

Comment:
what is the structure of all_scores? maybe a comment would help, but i dont really understand why we ++ if it's less than, return if its == or > than.